node/core/dispute-coordinator/src/spam_slots.rs

```diff
@@ -54,15 +54,15 @@ pub struct SpamSlots {
 }
 
 /// Unconfirmed disputes to be passed at initialization.
-pub type UnconfirmedDisputes = HashMap<(SessionIndex, CandidateHash), BTreeSet<ValidatorIndex>>;
+pub type UnconfirmedDisputes = HashMap<CandidateHash, (SessionIndex, BTreeSet<ValidatorIndex>)>;
```

Comment:
I think this is not good but I can't explain why very well (@eskimor,  help). All the code I have seen uses `(SessionIndex, CandidateHash)` as a key.

Why we need this btw?

Reply:
Reversed these changes. Was reminded of a saying. "If it ain't broke, don't fix it!"